src/test/java/htsjdk/samtools/seekablestream/SeekableBufferedStreamTest.java

```diff
@@ -123,6 +124,43 @@ public void testSkip() throws IOException {
         }
     }
 
+    @Test
+    public void testSeek() throws IOException {
+        int bufferSize = 20000;
+        int startPosition = 250000;
+        int length = 5000;
+
+        final int[] RELATIVE_SEEK_OFFSET = new int[]{-bufferSize*2, -bufferSize, -bufferSize/2, -length, -length/2, 0,
```

Comment:
it's weird that this is all caps generally style is to only use caps for static constants, but it does match the rest of the tests, so I guess keep it for consistency?

Reply:
Yes, I did this for consistency.